src/Common/nunit/Options.cs

```diff
@@ -349,17 +353,25 @@ public string[] GetValueSeparators ()
         protected static T Parse<T> (string value, OptionContext c)
         {
             Type tt = typeof (T);
+#if NETCORE
+            bool nullable = tt.GetTypeInfo().IsValueType && tt.GetTypeInfo().IsGenericType && 
+                !tt.GetTypeInfo().IsGenericTypeDefinition && 
+                tt.GetGenericTypeDefinition () == typeof (Nullable<>);
+            Type targetType = nullable ? tt.GetGenericArguments () [0] : typeof (T);
+#else
```

Comment:
Another agreement :smile: 

There is a lot we don't use and this has diverged so much from the original, it would be nice to clean it up.